mephisto/data_model/task_run.py

```diff
@@ -113,6 +113,27 @@ class TaskRunArgs:
         },
     )
 
+    post_build_script: str = field(
+        default="",
+        metadata={
+            "help": (
+                "The path to a shell script that will run right before npm build."
```

Comment:
This is confusing. We say right _before_ build in the description text, but the field is called _post_ build script. Should we have called it post-install script or pre-build script instead?